Classes/TcaDataGenerator/Generator.php

```diff
@@ -91,6 +91,9 @@ public function create()
                 'hidden' => 0,
                 'pid' => $neighborPage,
             ];
+            if ($mainTable === 'tx_styleguide_flex') {
+                $data['pages'][$newIdOfPage]['TSconfig'] = 'RTE.default.preset = minimal';
```

Comment:
this syntax should work, too: RTE.config.[tableName].[fieldName]  ... this way you can restrict the tsconfig to a specific fields (and thus all rte fields in there if that is a flex field, i guess)